packages/flutter/lib/src/cupertino/picker.dart

```diff
@@ -0,0 +1,188 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/widgets.dart';
+
+/// Color of the 'magnifier' lens border.
+const Color _kHighlighterBorder = const Color(0xFF7F7F7F);
+const Color _kDefaultBackground = const Color(0xFFD2D4DB);
+/// Eyeballed value comparing with a native picker.
+const double _kDefaultDiameterRatio = 1.1;
+/// A 255 based opacity value that hides the wheel above and below the 'magnifier' lens.
+const int _kForegroundScreenOpacity = 180;
+
+/// An iOS-styled picker.
+///
+/// Displays the provided [children] widgets on a wheel for selection and
+/// calls back when the currently selected item changes.
+///
+/// Can be used with [showModalBottomSheet] to display the picker modally at the
+/// bottom of the screen.
+///
+/// See also:
+///
+///  * [ListWheelScrollView], the generic widget backing this picker without
+///    the iOS design specific chrome.
+///  * <https://developer.apple.com/ios/human-interface-guidelines/controls/pickers/>
+class CupertinoPicker extends StatefulWidget {
+  const CupertinoPicker({
+    this.diameterRatio: _kDefaultDiameterRatio,
+    this.backgroundColor: _kDefaultBackground,
+    this.scrollController,
+    @required this.itemExtent,
+    @required this.onSelectedItemChanged,
+    @required this.children,
+  }) : assert(diameterRatio != null && diameterRatio > 0.0),
+       assert(backgroundColor != null),
+       assert(itemExtent != null && itemExtent > 0.0);
+
+  /// Relative ratio between this picker's height and the simulated cylinder's diameter.
+  ///
+  /// Smaller values creates more pronounced curvatures in the scrollable wheel.
+  ///
+  /// For more details, see [ListWheelScrollView.diameterRatio].
+  ///
+  /// Must not be null and defaults to `1.1` to visually mimic iOS.
+  final double diameterRatio;
+
+  /// Background color behind the children.
+  ///
+  /// Must not be null.
+  final Color backgroundColor;
```

Comment:
Can it be transparent? And if so, can't null be equivalent to that, but more efficient?